packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -73,8 +81,13 @@ class RouteSettings {
     this.isInitialRoute: false
   });
 
+  /// The name of the route (e.g., "/settings").
   final String name;
+
+  /// The set of keys that are most useful for constructoring [Hero] transitions.
```

Comment:
s/useful/relevant/. Maybe give an example, something about how if you have a list with many heroes with the same tag, then the most valuable key would be that of the hero from the row you're animating from.